crates/edr_eth/src/remote/request_methods.rs

```diff
@@ -0,0 +1,88 @@
+use revm_primitives::{Address, B256};
+
+use crate::{
+    remote::{eth::GetLogsInput, BlockSpec, PreEip1898BlockSpec},
+    U256,
+};
+
+/// Methods for requests to a remote Ethereum node. Only contains methods
+/// supported by the [`crate::remote::client::RpcClient`].
+#[derive(Clone, Debug, PartialEq, serde::Serialize)]
+#[serde(tag = "method", content = "params")]
+pub enum RequestMethod {
```

Comment:
Missing `feeHistory`

Reply:
Fixed in: [8c20657](https://github.com/NomicFoundation/hardhat/pull/4687/commits/8c20657c1125d4f3efba6b012f7b1d9c9d2889f3)